client/my-sites/post-share/index.jsx

```diff
@@ -51,11 +58,28 @@ class PostShare extends Component {
 		requestConnections: PropTypes.func
 	};
 
+	static FOOTER_SECTION_SCHEDULED = 'footer-section-scheduled';
+	static FOOTER_SECTION_PUBLISHED = 'footer-section-published';
+
 	state = {
 		message: PostMetadata.publicizeMessage( this.props.post ) || this.props.post.title,
 		skipped: PostMetadata.publicizeSkipped( this.props.post ) || [],
+		footerSection: PostShare.FOOTER_SECTION_SCHEDULED
+
 	};
 
+	constructor() {
+		super( ...arguments );
+
+		this.setFooterSection = this.setFooterSection.bind( this );
+	}
+
+	setFooterSection( section ) {
```

Comment:
Nice idea. However, we still need to use `partial` as we are passing the footer section. For example:
```
onClick={ partial(
	this.setFooterSection,
	PostShare.FOOTER_SECTION_PUBLISHED
) }
```